articles/openshift/oshift-faq.md

```diff
@@ -29,15 +32,15 @@ By offering this service, we take on all that complexity so that customers can i
 
 ### Is UKCloud for OpenShift a single-tenant or multi-tenant solution?
 
-UKCloud for OpenShift is built as a single-tenant platform on-top of UKCloud's secure, assured multi-tenant UKCloud for OpenStack IaaS platform, helping to deliver the benefits of single tenant isolation with the economics and flexibility of multi-tenant infrastructure.
+UKCloud for OpenShift is built as an isolated single-tenant platform on top of UKCloud's secure, assured multi-tenant UKCloud for OpenStack IaaS platform, helping to deliver the benefits of single-tenant isolation with the economics and flexibility of multi-tenant infrastructure.
 
 ### How is UKCloud for OpenShift billed?
 
 This service comprises of two main chargeable elements:
 
-  - **Foundation Pack -** provides the foundation OpenShift services to support the orchestration and management of the container runtime environments. Each customer environment requires only a single Foundation Pack
+  - **Foundation Pack -** providing an initial footprint of 32GiB of RAM, billed by the month with a one-month minimum commitment
```

Comment:
I'm still not happy with this bit. I've made a suggestion below though - but it needs to clarify that the foundation pack doesn't provide any resource to run applications on. Only the control plane and infrastructure services.

```suggestion
  - **Foundation Pack -** providing the cluster control plane and an initial footprint of 32GiB of RAM to provide the infrastructure services, billed by the month with a one-month minimum commitment. The addition of Runtime packs is required to provide the resource to run customer applications.
```